Assets/Prefabs/UI/StartSplashScreen/StartSplashScreen.anim

```diff
@@ -0,0 +1,377 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Pls create Animation folder in assets folder and move this file to created folder.